packages/twenty-docs/fr/developers/self-hosting/docker-compose.mdx

```diff
@@ -17,7 +17,7 @@ Ce guide fournit des instructions pas-à-pas pour installer et configurer l'appl
 
 **Important :** Modifiez uniquement les paramètres explicitement mentionnés dans ce guide. Modifier d'autres configurations peut entraîner des problèmes.
 
-Consultez les documents [Configurer les Variables d’Environnement](https://docs.twenty.com/developers/self-hosting/setup) pour la configuration avancée. Toutes les variables d'environnement doivent être déclarées dans le fichier docker-compose.yml au niveau du serveur et/ou du travailleur, selon la variable.
+Consultez les documents [Configurer les Variables d’Environnement](https://docs.twenty.com/fr/developers/self-hosting/setup) pour la configuration avancée. Toutes les variables d'environnement doivent être déclarées dans le fichier docker-compose.yml au niveau du serveur et/ou du travailleur, selon la variable.
```

Comment:
This is not a sustainable solution because it will be overwritten when translations, we need to switch to relative links or do something smarter I think

Reply:
Is it the recommended way of doing things? I feel like we can do something cleaner no?

Reply:
I can merge this for now but curious to have your feedback/recommandations